apps/web/src/store/message-cache.ts

```diff
@@ -0,0 +1,50 @@
+import type { ConversationExport } from '@xmtp/xmtp-js/dist/types/src/conversations/Conversation';
+import { create } from 'zustand';
+import { persist } from 'zustand/middleware';
+
+// If any breaking changes to the ConversationExport schema occur, increment the cache version.
+const CONVERSATION_CACHE_VERSION = 1;
+
+/**
+ * The MessageCache is a JSON serializable Zustand store that is persisted to LocalStorage
+ * Persisting conversations to the cache saves on both bandwidth and CPU cycles, as we don't have to re-fetch or re-decrypt conversations on subsequent page loads
+ */
+interface MessageCache {
```

Comment:
The name `MessageCache` feels a little misleading, seeing as it doesn't store any messages

Reply:
makes sense, addressed here: https://github.com/lensterxyz/lenster/pull/1908/commits/2b5d5f8f34642b7198e651786638d744c9fbdb7e